.vscode/launch.json

```diff
@@ -0,0 +1,43 @@
+{
+  // Use IntelliSense to learn about possible Node.js debug attributes.
```

Comment:
I believe we do not need to track any editor specific scripts

Reply:
Removed this file